.gitignore

```diff
@@ -1,3 +1,12 @@
 *.swp
 *.~
 *.py[co]
+__pycache__/
+.idea/
+nohup.out
+.gcloudignore
+cloud_sql_proxy
+.vscode/
+.fuse_hidden*
```

Comment:
What's this?

Reply:
These files generate when we delete a file which is still in use. Will remove it.